src/styles/class-overrides.scss

```diff
@@ -0,0 +1,128 @@
+.usa-prose,
+.usa-intro,
+.font-heading-xs,
+.font-heading-xl,
+.text-primary,
+.usa-graphic-list__heading,
+.usa-hero__heading {
+  font-family: family("ui");
+}
+
+.usa-nav__submenu,
+.usa-nav__primary button[aria-expanded="true"],
+.usa-hero__callout,
+.usa-section--dark {
+  background-color: color("base-darker");
+}
+
+.usa-nav__primary button[aria-expanded="false"] {
+  color: color("disabled-dark");
+  &:hover {
+    color: color("base-darker");
+  }
+  &::after {
+    border-style: solid;
```

Comment:
Same here, is it possible to organize this? For instance, in alphabetical order? Is there a convention you can follow?